### PR TITLE
[Rename] remove remaining references

### PR DESCRIPTION
**Pull request by kavilla**
### Description
Some values were added but should not impact current cuts of the beta. For example, just maps legacy where the incorrect map config was used. This is not used when running main.

Signed-off-by: Kawika Avilla <kavilla414@gmail.com>
 
### Issues Resolved
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/221
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/legacy/server/config/schema.js [hunk 1]
```diff
@@ -205,11 +205,11 @@ export default () =>
       opensearchManifestServiceUrl: Joi.string().default(
         'https://maps.search-services.aws.a2z.com/v4/ap-southeast-1/manifest'
       ),
-      emsFileApiUrl: Joi.string().default('https://vector.maps.elastic.co'),
-      emsTileApiUrl: Joi.string().default('https://tiles.maps.elastic.co'),
-      emsLandingPageUrl: Joi.string().default('https://maps.elastic.co/v7.9'),
+      emsFileApiUrl: Joi.string().default('https://vector.maps.opensearch.org'),
+      emsTileApiUrl: Joi.string().default('https://tiles.maps.opensearch.org'),
+      emsLandingPageUrl: Joi.string().default('https://maps.opensearch.org/v7.10'),
```
Comment (boktorbb-amzn) [inline src/legacy/server/config/schema.js:210]: Are these dead links? Does it affect the rendering in any way?

Comment (kavilla) [reply src/legacy/server/config/schema.js:210]: > Are these dead links?
Yes but it removes the elastic.co stuff

> Does it affect the rendering in any way?
It uses `opensearchManifestServiceUrl` to actual get the map values. To be honest, there can be additional work that cleans up unneeded stuff.

Comment (kavilla) [reply src/legacy/server/config/schema.js]: I just updated those values to real links. Better to be unused and real rather than used and fake.

**Review by ananzh — APPROVED** [review]
Nice

**Pull request merged**